jobs/service-fabrik-broker/spec

```diff
@@ -16,217 +16,147 @@ templates:
   config/syslog_forwarding.conf.erb: config/syslog_forwarding.conf
   config/circuit-breaker-config.yml.erb: config/circuit-breaker-config.yml
 
+provides:
+- {name: broker, type: broker, properties: [name, username, password, skip_ssl_validation, log_level, enable_circuit_breaker, feature.ServiceInstanceAutoUpdate, internal.port, internal.ip, internal.ssl,internal.log_event, internal.domain_socket.path, external.host, external.log_event, external.api_requires_admin_scope, mongodb.url, mongodb.provision.plan_id, mongodb.provision.network_index, mongodb.deployment_name, mongodb.retry_connect.max_attempt, mongodb.retry_connect.min_delay, mongodb.record_max_fetch_count, mongodb.backup.schedule_interval, mongodb.agent.provider.container, mongodb.backup.schedule_interval, mongodb.deployment_name, mongodb.agent.provider.container, monitoring.include_response_body, monitoring.events_logged_in_db, monitoring.unauthorized.include_response_body, riemann.show_errors, riemann.log_additional_event, riemann.skip_http_status_40x, cf.url, cf.username, cf.password, backup.retention_period_in_days, backup.max_num_on_demand_backup, backup.status_check_every, backup.retry_delay_on_error, backup.lock_check_delay_on_restart, backup.backup_restore_status_poller_timeout, backup.backup_restore_status_check_every, backup.abort_time_out, backup.provider, docker.url, docker.skip_ssl_validation, docker.allocate_docker_host_ports, docker.volume_driver, common.tls_cacert, common.tls_client_cert, common.tls_client_key, riemann.host, riemann.port, riemann.prefix, riemann.metrics, syslog.host, syslog.port, sys_log_level, event.defaults.http_success_codes, event.defaults.http_inprogress_codes, event.defaults.log_inprogress_state, event.defaults.cf_last_operation, event.defaults.ignore_service_fabrik_operation, event.defaults.include_response_body, event.delete.http_success_codes, quota, services, directors ]}
```

Comment:
The properties list is quite huge. Can you please check if providing just the top level names is sufficient. for ex., if you just specify `mongodb`, i guess all properties under it will be automatically exported or should get exported. By doing so, reduces your properties list considerably.
Refer [here in this sample](https://github.com/jagadish-kb/service-fabrik-blueprint-boshrelease/blob/1c16e9e5ca5e2935407b925ae478fb6bf8460281/jobs/blueprint/spec#L19) - only top level admin attrib is exported, but admin has userid, password defined in from [manifest](https://github.com/jagadish-kb/service-fabrik-broker/blob/feature/bosh2.0/config/templates/blueprint-manifest.yml.ejs#L49) & they are [consumed](https://github.com/jagadish-kb/service-fabrik-blueprint-boshrelease/blob/1c16e9e5ca5e2935407b925ae478fb6bf8460281/jobs/blueprint-agent/templates/config/settings.yml.erb#L34) as well by referring just top level element.